Neos.Neos/Classes/Service/HtmlAugmenter.php

```diff
@@ -62,7 +62,8 @@ public function addAttributes($html, array $attributes, $fallbackTagName = 'div'
      */
     protected function getHtmlRootElement($html)
     {
-        if (trim($html) === '') {
+        $html = trim($html);
```

Comment:
The `HtmlAugmenter` now always strips its content, I'm not sure if this might have unwanted side effects?